protobuf/src/test/java/io/grpc/protobuf/StatusProtoTest.java

```diff
@@ -176,6 +176,15 @@ public void fromThrowable_shouldReturnNullIfNoEmbeddedStatus() {
     assertNull(StatusProto.fromThrowable(nestedSe));
   }
 
+  @Test
+  public void toStatusExceptionWithMetadataAndCause_shouldCaptureCause() {
+    RuntimeException exc = new RuntimeException("This is a test exception.");
```

Comment:
probably rename `exc` -> `cause`

Reply:
Yeah, `ex`, `exception`, `cause`, etc would be more common, but we can live with this.